datarepo-clienttests/src/main/java/scripts/testscripts/DRSLookup.java

```diff
@@ -43,6 +43,13 @@ public DRSLookup() {
   private static List<BlobId> scratchFiles = new ArrayList<>();
   private String dirObjectId;
 
+  private Integer K = 1;
```

Comment:
`K` is not very descriptive. How about
```suggestion
  private Integer NUM_DRS_LOOKUPS = 1;
```